src/askContainer/webViews/simulateSkillWebview.ts

```diff
@@ -31,15 +28,15 @@ export class SimulateSkillWebview extends AbstractWebView {
     getHtmlForView(): string {
         Logger.debug(`Calling method: ${this.viewId}.getHtmlForView`);
 
-        const webview: vscode.Webview = this.getWebview() as vscode.Webview;
-        const simulateCss: vscode.Uri = webview.asWebviewUri(
+        //const webview: vscode.Webview = this.getWebview() as vscode.Webview;
```

Comment:
Any reason this is commented out?

Reply:
Thanks to point out, Line#31 is useless now, can remove it. Before this,  we use the commented line to get webview, then call `asWebviewUri`, but it's not straightforward, so change to use the way as line#32.